fxa-oauth-server/config/dev.json

```diff
@@ -134,6 +134,16 @@
       "redirectUri": "urn:ietf:wg:oauth:2.0:oob",
       "trusted": true,
       "canGrant": true
+    },
+    {
+      "id": "3c49430b43dfba77",
+      "name": "Android Components Reference Browser",
+      "hashedSecret": "a7ee3482fab1782f5d3945cde06bb911605a8dfc1a45e4b77bc76615d5671e51",
+      "redirectUri": "https://XXX.TODO.WHAT.GOES.HERE",
```

Comment:
Can you change this to `https://accounts.firefox.com/oauth/success/3c49430b43dfba77`?